plugins/SitesManager/Controller.php

```diff
@@ -259,4 +263,19 @@ private function getActiveTabOnLoad($templateData)
 
         return $tabToDisplay;
     }
+
+    private function getInviteUserLink()
+    {
+        if (!Piwik::hasUserSuperUserAccess()) {
+            return 'https://matomo.org/faq/general/manage-users/#imanadmin-creating-users';
```

Comment:
Should we show this for admin users? They actually would be allowed to invite new users...
Also the anchor of the link actually doesn't seem to work.

Reply:
@sgiehl the link is opening for me as expected

Reply:
yes we should change it to admin

Reply:
Changed to admin